schemas/tls.yml

```diff
@@ -0,0 +1,174 @@
+---
+- name: tls
+  title: TLS
+  group: 2
+  short: Fields describing a TLS connection.
+  description: >
+    Fields related to TLS activity.
+  type: group
+  fields:
+
+    - name: version.protocol
+      level: extended
+      type: keyword
+      description: Normalized protocol name parsed from original string (e.g. ssl, tls).
+
+    - name: version.number
+      level: extended
+      type: keyword
+      description: Numeric part of the version parsed from the original string (e.g. 1.2, 3).
+
+    - name: cipher
+      type: keyword
+      level: extended
+      description: String indicating the cipher used during the current connection.
+
+    - name: curve
+      type: keyword
+      level: extended
+      description: String indicating the curve used for the given cipher, when applicable.
+
+    - name: resumed
+      type: boolean
+      level: extended
+      description: Boolean flag indicating if this TLS connection was resumed from an existing TLS negotiation.
+
+    - name: next_protocol
+      type: keyword
+      level: extended
+      description: String indicating the protocol being tunneled (e.g. http/1.1, spdy/3, imap, webrtc)
+
+    - name: client.ja3
+      type: keyword
+      level: extended
+      description: A hash that identifies clients based on how they perform an SSL/TLS handshake.
+
+    - name: client.server_name
+      type: keyword
+      level: extended
+      description: >
+        Also called an SNI, this tells the server which hostname to which the client is attempting to connect.
```

Comment:
We should likely call out explicitly that `tls.client.server_name`, when available, should be copied to `destination.domain`.

Reply:
Does it make sense to add a field for SAN specification? As the SNI stuff kind of implies, there are going to be plenty of times where the CN (`server.subject` here I believe) doesn't match a SAN specified in the cert.